code/modules/food_and_drink/snacks.dm

```diff
@@ -2156,14 +2156,59 @@
 // boy i wish byond had static members doop doop
 var/list/valid_jellybean_reagents = childrentypesof(/datum/reagent)
 
-//#ifdef HALLOWEEN
-/obj/item/reagent_containers/food/snacks/candy/everyflavor
-	name = "\improper Farty Snott's Every Flavour Bean"
-	desc = "A favorite halloween sweet worldwide!"
+/obj/item/reagent_containers/food/snacks/candy/jellybean/
+	name = "jelly bean"
+	desc = "YOU SHOULDN'T SEE THIS OBJECT"
 	icon_state = "bean"
 	amount = 1
 	initial_volume = 100
 	sugar_content = 0 // hacky, I know. but it's necessary for the color!
+	var/flavor
+	var/phrase
+	var/tastesbad = 0
```

Comment:
This var seems to be implemented backwards, 0 for true and 1 for false.

Reply:
shit youre right, committing fix now